qiskit/algorithms/__init__.py

```diff
@@ -240,6 +241,16 @@
    AlgorithmError
 
 
+Trotterization-based Quantum Real Time Evolution
+------------------------------------------------
+
+Package for primitives-enabled Trotterization-based quantum time evolution algorithm - TrotterQRTE.
+
+.. autosummary::
+   :toctree: ../stubs/
+
+   time_evolvers.trotterization
+
```

Comment:
Would this not be better for users if this were more co-located with the TimeEvolvers and the interfaces above so the related content/function is more together. I imagine PVQD will be there and the variational ones too.

The init file from that location is now included in the docs via this. However the time evolvers TrotterQRTE is still missing from docs

![image](https://user-images.githubusercontent.com/40241007/191355513-2e1c0e9a-3c97-4b83-94f3-f7869e965dcb.png)

Reply:
I started having an impression that the order in this file was supposed to be alphabetical. I tried to follow that. However, the order has already been broken by 1 or 2 previous additions. In this case, I will follow your suggestion to keep things co-located.